elements.py

```diff
@@ -31,53 +28,57 @@ def __init__(self):
         self.c = 0
         self.playflag = 0
 
-        if(self.x == 390 or self.x == 510):
+        if (self.x == 390 or self.x == 510):
```

Comment:
Please remove the parentheses here and every other place where it's redundant.

Reply:
Fixed